### PR TITLE
Fixed #65

### LINKED ISSUE #65: the projector should display the final result, with time slots and location

### PR DESCRIPTION
**Pull request by franc6**
Moved functionality of src/routes/projector.php to a new class in src/classes/Projector.php.

Added code in new Projector class to show a series of tables, one for each round of BOFs.  The table will display session, room, facilitator, and vote information for the round.

### CODE REVIEW

**Comment by franc6** [comment]
Argh; latest change was NOT tested -- I missed the error message when copying the file to the server. :(  I also noticed a few things to clean up.  Please wait until further comment before reviewing.

**Comment by franc6** [comment]
OK, now it's ready for review.

File: src/templates/proj_layout.html [hunk 1]
```diff
@@ -3,14 +3,18 @@
     <head>
         <meta charset="UTF-8">
         <title>{% block title %}ICCM BOF app{% endblock %}</title>
-        <link rel="stylesheet" href="/assets/css/main.css">
-	    <meta http-equiv="refresh" content="5" />
+        <link rel="stylesheet" href="/assets/css/{{branding}}.css">
+	    <meta http-equiv="refresh" content="500" />
```
Comment (tpokorra) [inline src/templates/proj_layout.html]: 500 seconds for the refresh? that is more than 8 minutes.
I agree 5 seconds is perhaps too frequent. Maybe change to 60 seconds?

File: src/lang/de.php [hunk 2]
```diff
@@ -35,5 +35,7 @@
     'Add Slot' => 'Slot Hinzufügen',
     'Rooms' => 'Zimmer',
     'Add Room' => 'Zimmer Hinzufügen',
+    'Room' => 'Zimmer',
```
Comment (tpokorra) [inline src/lang/de.php]: Thinking more about it, Raum is probably better than Zimmer. Zimmer is part of an appartment or a house where you live. Raum is more general, and is used for meeting rooms and halls.

**Comment by franc6** [comment]
Good catch!  I forgot I changed it. I made the change so I would have time to modify the styles in the browser to test the updates before putting them in the style sheet. I thought that would be a big enough difference for me to catch it and put it back before committing. Obviously, I still missed it. :(

I’ll change it back to 5, since that was my intent. :)

TOM

-- 
Preserve wildlife -- pickle a squirrel!

**Comment by franc6** [comment]
I’ll make that change here, too.

**Pull request merged**